ext/opcache/ZendAccelerator.c

```diff
@@ -1813,8 +1813,16 @@ static zend_persistent_script *opcache_compile_file(zend_file_handle *file_handl
 		CG(compiler_options) |= ZEND_COMPILE_NO_CONSTANT_SUBSTITUTION;
 		CG(compiler_options) |= ZEND_COMPILE_IGNORE_OTHER_FILES;
 		CG(compiler_options) |= ZEND_COMPILE_IGNORE_OBSERVER;
+#ifdef ZEND_WIN32
+		/* On Windows, don't compile with internal classes. Shm may be attached from different
+		 * processes with internal classes living in different addresses. */
+		CG(compiler_options) |= ZEND_COMPILE_IGNORE_OBSERVER;
```

Comment:
You probably mean ``ZEND_COMPILE_IGNORE_INTERNAL_CLASSES``

Reply:
Sorry, stupid copy/paste error.